ootd/views.py

```diff
@@ -1,3 +1,310 @@
+        try:
+            posts = Ootd.objects.select_related('user').prefetch_related(
+                'like_set',
+                'ootdimageurl_set',
+                'comment_set',
+                'product_set',
+                'tag',
+            )
+
+            post = posts.get(id = ootd_id)
+            ootd_post = {
+                    'contentImg'        : [image.image_url for image in post.ootdimageurl_set.all()],
+                    'productImg'        : [product_image.main_image_url for product_image in post.product_set.all()],
+                    'productName'       : [product_name.title for product_name in post.product_set.all()],
+                    'price'             : [product_price.price for product_price in post.product_set.all()],
+                    'sale'              : [product_sales.sales_product for product_sales in post.product_set.all()],
+                    'authorImg'         : post.user.profile_image_url,
+                    'author'            : post.user.nickname,
+                    'introdution'       : post.user.description,
+                    'tagsName'          : [tag_name.tag.tag_name for tag_name in post.ootdtag_set.all()],
+                    'datetime'          : post.created_at.strftime("%Y-%m-%d %H:%M:%S") ,
+                    'description'       : post.description,
+                    'follower'          : post.like_set.count(),
+                    'commentNum'        : post.comment_set.count(),
+                    'comments'          : [
+                            {
+                                'commentAuthor'    : comment.user.nickname,
+                                'commentAuthorImg' : comment.user.profile_image_url,
+                                'comment'          : comment.content,
+                                'parent_id'        : comment.parent_id 
+                            } for comment in post.comment_set.all()
+                                        ]
+                        }
+                
+            return JsonResponse(ootd_post, status = 200)
+        
+        except Ootd.DoesNotExist:
+            return JsonResponse({"MESSAGE" : "OOTD_DOES_NOT_EXIST"}, status = 400)
+
+class OotdlView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            user = User.objects.get(id = data['user_id'])
+            post = Ootd.objects.create(
+                description = data['description'],
+                user = user
+            )
+            # 태그 로직 (# 기준으로 split하여서 태그인 값만 걸러 중간 테이블에 추가)
+            description_list = data['description'].split('#')
+            for string in description_list[1:]:
+                if string[0] == ' ':
+                    continue
+                tag = string.split()[0]
+
+                tag =Tag.objects.get_or_create(tag_name=str('#'+tag))
+                post.tag.add(tag)
+
+                OotdTag.objects.create(ootd =post, tag = tag)
```

Comment:
```suggestion
                OotdTag.objects.create(ootd =post, tag = tag)
```
이코드가 남아 있으면 중복생성 될것 같은데 아닌가요? 확인 부탁드립니다.

Reply:
중복으로 코드를 짠 것 같습니다... 삭제 하겠습니다!